library/pages/components/ExperienceExamplesPage/util.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ComponentExampleCard from '../../../styleguideComponents/ComponentExampleCard';
+
+export default function Util(props) {
+    const { response, type } = props;
+
+    if (response && Array.isArray(response.results) && response.results.length) {
+        if (type === 'experience-examples') {
```

Comment:
there is not need for this check since here always it will be experience-examples